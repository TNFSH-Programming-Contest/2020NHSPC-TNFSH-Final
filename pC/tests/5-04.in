99 99 31
31 89 72898
89 86 881125
52 86 97048
89 77 985348
98 89 982833
89 79 868701
89 19 173633
89 17 309327
89 4 151758
35 89 256939
89 43 244546
15 89 196963
94 89 180545
89 8 936023
40 89 947943
89 72 124320
23 89 14587
68 89 584305
58 89 785606
89 57 311785
61 89 952133
74 89 11562
47 89 728713
13 89 65850
89 14 627020
89 9 158440
92 89 464630
89 54 105481
89 12 713560
89 88 542379
89 24 166369
81 89 761405
95 89 22538
89 97 678696
89 83 452473
89 78 522818
21 89 987688
82 89 876947
11 89 418716
89 36 636242
89 49 515719
89 70 6435
73 89 25486
89 45 283727
93 89 163454
89 60 578958
50 89 658209
89 32 737366
89 27 361039
25 89 430055
89 65 851270
89 91 765521
44 89 189419
3 89 988263
89 20 193360
89 64 235444
84 89 991928
89 56 60337
89 48 70717
33 89 789096
33 22 178166
80 89 950845
89 34 269932
26 89 353147
59 89 805980
71 89 807052
53 89 856768
89 66 557561
76 89 316703
89 18 987461
38 89 852381
89 30 63099
46 89 209282
89 1 783875
2 89 717561
55 89 88202
96 89 420620
89 16 859469
89 51 446792
89 10 15747
89 67 620680
89 37 280891
89 87 293240
87 63 559273
89 28 365630
41 89 194235
89 7 904041
6 89 650528
42 89 378662
89 62 367460
89 69 39387
89 5 639933
89 39 633345
89 85 971777
89 75 383126
89 90 959580
89 29 762463
89 99 741565
94 75 870279
