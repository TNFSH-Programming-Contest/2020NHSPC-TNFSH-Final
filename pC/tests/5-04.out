1713546 1580918 2122322 449312 1425662 1446852 1953878 2017842 462676 177290 983228 1572916 277496 1399836 539722 1864734 764450 2120718 493062 532516 2121172 3120480 174970 478534 1005906 852090 867874 877056 1670722 271994 0 1620528 1723988 685660 659674 1418280 707578 1850558 1412486 2041682 534266 903120 634888 524634 713250 564360 1603222 287230 1177234 1462214 1039380 3153213 1859332 356758 322200 266470 769366 1717008 1757756 1303712 2050062 880716 2776233 616684 1848336 1260918 1387156 1314406 224570 158666 1759900 394436 196768 168920 912048 779202 2116492 1191432 1883198 2047486 1668606 1899690 1050742 2129652 2089350 1908046 732276 1230554 72898 2064956 1676838 1075056 472704 506886 190872 987036 1503188 2111462 1628926 